### DISCUSSION TITLE
How to lazy load locale messages from external url ? [Nuxt 3/i18n V8]

### DISCUSSION THREAD

**Discussion by devhus** [nuxt-modules/i18n#1749] [Q&A]
Hello, I am wondering how to import messages from an external URL.
For example, the nuxt app is on domain1.com and the messages files exist on domain2.com/locales/locale.json

I have tried the ``lazy`` option turned to ``true`` along with an async function like the one mentioned in the link below but the messages are not even loading from the Promise and so cannot fetch from an external URL within that promise.
https://i18n.nuxtjs.org/lazy-load-translations/
```js
export default async (context, locale) => {
  return await Promise.resolve({
    welcome: 'Welcome'
  })
}
```
Any idea what to do?

Using the latest version of nuxt & the library 
nuxt 3.0.0
i18n v8.0.0-beta.7

**Comment by ineshbose** [comment]
Hi, what version of Nuxt and `@nuxtjs/i18n` are you looking to use?

**Reply by devhus** [reply]
nuxt 3.0.0
i18n v8.0.0-beta.7

**Reply by yplian** [reply]
You should probably take a look at the new version of the document, which is currently not supported.
[file is still not supported function , Promises](https://v8.i18n.nuxtjs.org/guide/lazy-load-translations)
![image](https://user-images.githubusercontent.com/23043772/208571236-5f4c5af7-4a7c-4749-ad7e-6a20afdf1663.png)

**Reply by devhus** [reply]
Oh, I thought that was the documentation 
https://i18n.nuxtjs.org/lazy-load-translations
![image](https://user-images.githubusercontent.com/58523735/209147567-2e8fb2f0-fee1-4ffb-b08a-94568b4495d0.png)

**Reply by plcdnl** [reply]
Hi @devhus, how you think to bypass this problem temporarily?
